extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -487,11 +487,15 @@ angular.module('PaperUI.controllers.configuration', []).controller('Configuratio
     };
 
     function checkAdvance(channels) {
-        angular.forEach(channels, function(value) {
-            if (value.advanced) {
-                return true;
+        if (channels) {
+            for (var i = 0, len = channels.length; i < len; i++) {
+                var channel = channels[i];
+                var channelType = $scope.getChannelTypeById(channel.id);
+                if (channelType.advanced) {
```

Comment:
No, it is perfectly fine to have channels without a type - so you might want to adapt your PR as well accordingly.